Wrappers.h

```diff
@@ -0,0 +1,40 @@
+#include "Shoe.cpp"
+#include "Wurlitzer.cpp"
+#include "ElectricBass.cpp"
+#include "Band.cpp"
+#include "Clothing.cpp"
```

Comment:
NEVER include `cpp` files. 
you need to tell the compiler to compile these files.
- [ ] revise